extensions/ql-vscode/src/remote-queries/remote-queries-markdown-generation.ts

```diff
@@ -144,22 +144,67 @@ function highlightCodeLines(
   return `${partiallyHighlightedLine.plainSection1}<strong>${partiallyHighlightedLine.highlightedSection}</strong>${partiallyHighlightedLine.plainSection2}`;
 }
 
-function buildMarkdownAlertMessage(interpretedResult: AnalysisAlert): string {
+function buildMarkdownAlertMessage(
```

Comment:
What do you think about appending a `For` to these new `buildMarkdown` functions e.g. `buildMarkdownForAlertMessage`? Or, to match other function names in this module, you could call them `generateMarkdownForXXX`